tests/render_ok.rs

```diff
@@ -12,6 +12,7 @@ use common::{Product, Review, read_file};
 
 fn assert_template_ok(path: &str, others: Vec<&str>) {
     let mut tera = Tera::default();
+    tera.autoescape_on(vec!["html"]);
```

Comment:
Since it worked, this means no template is actually testing this. Should probably add a template that needs escaping.